setup.py

```diff
@@ -32,8 +32,7 @@
     install_requires=[
             'pip>=21',
             'importlib-metadata<=4.12.0; python_version <= "3.7"', # problem with gym for importlib-metadata==5.0.0 and python <=3.7
-            'wheel==0.38.4', # Required until we can upgrade to gym >= 0.22.0
-            'setuptools',
+            'setuptools<=66.0.0', # Required until we can upgrade to gym >= 0.22.0
```

Comment:
@discordianfish So, to recap:
- `gym<=0.21` is not compatible with `wheel`+`setuptools` latest versions, more specifically:
  - `wheel>0.38.3.`
  - `setuptools>66.0.0`
- New `wheel` packages raise an error that:
  - In CICD is catched and make the pipeline fail
  - In local dev envs is ignored (but shown) and make the installation succeed
- New `setuptools` packages make the installation fail in both CICD and local dev envs

Considering that:
- Pinning `wheel` in `diambra-arena` requirements does not solve the CICD issue as wheel is needed before the package installation, thus we need to add the specific wheel package anyways
- New version of `wheel` does not prevent the package to be installed in local envs, but it shows an error message

Here I am only pinning `setuptools` package, as this solves the issue until we upgrade gym version.

Do you suggest to also pin wheel even if it does not make the installation fails in local envs to avoid showing the error?

Reply:
No this is perfect. Also thanks a lot for the detailed explanation!